ChangeLog

```diff
@@ -1,3 +1,17 @@
+0.11
+---
+
+* Not yet released.
+* Added support for upload command.
+
+
+0.10
+---
+
+* Not yet released.
+* Switched to requests
+
+
```

Comment:
There is no need to make minor version with every commit, both changes will go to 0.10.